controllers/productController.js

```diff
@@ -0,0 +1,15 @@
+const productService = require("../services/productService");
+
+const detailProduct = async (req, res) => {
+  try {
+    const { productId } = req.params;
+    const productID = await productService.getDetailproduct(productId);
+    return res.status(201).json({ data: productID });
```

Comment:
응답을 보낼 때 적절한 상태코드를 사용해 주세요.

```suggestion
    return res.status(200).json({ data: product });
```